go

```diff
@@ -80,6 +81,20 @@ DAEMONS = {
 
 NEEDS_SSH = %w(team-api)
 
+REMOTE_HOST = 'ubuntu@hub.18f.gov'
+REMOTE_ROOT = 'knowledge-sharing-toolkit'
+SECRETS_BUNDLE_NAME = '18f-knowledge-sharing-toolkit-secrets'
+SECRETS_BUNDLE_FILE = "#{SECRETS_BUNDLE_NAME}.tar.bz2"
+SECRET_FILES = %w(
+  */config/env-secret.sh
+  nginx/config/auth/pages-passwords.txt
+  nginx/config/ssl/dhparam*
+  nginx/config/ssl/keys/*
+  pages/config/pages.secret
+  ssh/config/id_rsa*
+  ssh/config/known_hosts*
+)
```

Comment:
The different types of config files here seem very inconsistent...can we move everything to be configured the same way? Fine if you want to break this out to a separate Issue.